java-checks/src/test/files/checks/JacksonDeserialization.java

```diff
@@ -0,0 +1,58 @@
+import com.fasterxml.jackson.annotation.JsonTypeInfo;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import static com.fasterxml.jackson.annotation.JsonTypeInfo.Id.MINIMAL_CLASS;
+
+class JacksonDeserialization {
```

Comment:
you only have annotated classes in this test - please add tests for annotated fields as well